vuepress/docs/next/tutorials/samples/customer-portal.md

```diff
@@ -1,125 +1,166 @@
+
+### JIRA Service Management 
+
+The administrator begins at Jira Service Management to create users, projects, define the organization, and configure the service ticket system.
 
-A. **Define the Realm Setting**. The Realm is a set of users, credentials, roles, and groups. A user belongs to and logs into a Realm. 
+Go to Customers to add organizations and projects. Once added, click on the name of the organization to get the ID from the URL which is needed later. \
+ e.g. example.com/jira/servicedesk/projects/ECS/organization/3 → the organization ID is “3”
 
-B. **Create Roles**\
-You must select the roles available to users and/or groups. Create new roles with varying degrees of access for the needs of your application. You can also use the default roles by clicking on Client Roles and choosing entandodemo-customerportal-server. The following define their parameters for access:
-* cp-customer -  assigned directly to specific projects for a single customer
-* cp-partner - assigned directly to specific projects for multiple customers
-* cp-support -  read only view of all customer projects 
-* cp-admin - admin access for Customer Portal.
+### Entando Identity Management System 
+Logging into the Entando Identity Management System, you will see the landing page shown below. Like the App Builder, the left navigation bar is your guide for managing users, groups, and most importantly, roles. Using the RBAC model, define what access users have by the roles and groups they are assigned. Some important guidelines are noted below. 
+
+![Entando Identity Management System](./images/custportal-idmanagement-main.png)
 
-C. **Create New Users**:
-1. The user must be created in Jira first, then configured here for access to the Portal.
-2. From the navigation bar, go to Users. Click `Add User` at right.
-3. Enter the fields as needed but note the requirements for the following:\
-       `Username`: A unique name\
+* **Define the Realm Setting**. \
+The `Realm` is a set of users, credentials, roles, and groups. A user belongs to and logs into a `Realm`. 
+
+* **Create Roles**\
+You can  use the default roles by clicking on `Client Roles` and choosing `entandodemo-customerportal-server`. Access for each role is defined as follows:
+    * `cp-customer` -  assigned directly to specific projects for a single customer
+    * `cp-partner` - assigned directly to specific projects for multiple customers
+    * `cp-support` -  read only view of all customer projects
+    * `cp-admin` - admin access for the Customer Portal
+
+* **Create New Users**: \
+For a new user, you must create them first in Jira Service Management to integrate them into  your application. Then they are created and configured in the Entando ID Management System for access to the Customer Portal.
```

Comment:
awkward phrasing... what about: "A user must be created in Jira Service Management before they can be integrated into your application." ?